src/engine/client/cg_api.h

```diff
@@ -86,9 +86,30 @@ enum class MouseMode
 	SystemCursor, // The input is sent as positions, the cursor should be rendered by the system
 };
 
+#define MAX_MARK_FRAGMENTS 128
+#define MAX_MARK_POINTS 384
+
+struct fragmentPacket_t
+{
+	std::array<std::array<float, 3>, 4> originalPoints;
+	std::array<float, 3> projection;
+};
+
+struct fragmentBuffer_t
+{
+	std::array<std::array<float, 3>, MAX_MARK_POINTS> markPoints;
+	std::array<markFragment_t, MAX_MARK_POINTS> markFragments;
```

Comment:
Is this supposed to be MAX_MARK_FRAGMENTS?

Reply:
Right.